include/fiction/algorithms/simulation/sidb/occupation_probability_excited_states.hpp

```diff
@@ -0,0 +1,118 @@
+//
+// Created by Jan Drewniok on 06.02.23.
+//
+
+#ifndef FICTION_OCCUPATION_PROBABILITY_EXCITED_STATES_HPP
```

Comment:
Is this the occupation probability *of* excited states? Or *for*? Or something else? Maybe a preposition in the file and function name could help?